examples/sql/sql_example.py

```diff
@@ -86,13 +91,20 @@ def __repr__(self):
         key = row.get_object("__key")
         age = row.get_object("age")
 
+        # Rows can also be used similar to dictionaries
+        # with string column names
+
+        # key = row["__key"]
+        # age = row["age"]
+
```

Comment:
I think we can move this to line 48 which is the first get_object() call, it can be overlooked here. Optionally, we can write this comment multiple times

There is another a couple of get_object calls at line 58 and 59

Reply:
I didn't want to put these comments everywhere and this code sample is meant to be read from beginning to end to understand different ways to use SQL service. So, I think it it okay to leave it like this